src/kubernetes-ingress-controller/references/prometheus.md

```diff
@@ -11,7 +11,8 @@ This document is a reference for the former type.
 
 | Metric name | Description |
 |-------------|-------------|
-| `ingress_controller_configuration_push_count[success=true|false][protocol=db-less|deck]` | Count of successful or failed configuration pushes to Kong. <br><br> `protocol` describes the configuration protocol in use, which can be `db-less` or `deck`. <br><br> `success` logs the status of configuration updates. If `success` is `false`, an unrecoverable error occurred.  If `success` is `true`, the push succeeded with no errors.  |
+| `ingress_controller_configuration_push_count[success=true|false][protocol=db-less|deck]{% if_version gte:2.7.x inline:true %}[failure_reason=conflict|network|other]{% endif_version %}` | Count of successful or failed configuration pushes to Kong. <br><br> `protocol` describes the configuration protocol in use, which can be `db-less` or `deck`. <br><br> `success` logs the status of configuration updates. If `success` is `false`, an unrecoverable error occurred.  If `success` is `true`, the push succeeded with no errors. {% if_version gte:2.7.x inline:true %}<br><br> `failure_reason` is populated in case of `success="false"` and describes the reason of failure: <br> * `conflict` - configuration conflict that requires manual fix, <br> * `network` - network related issues (e.g. Kong being offline), <br> * `other` - other issues (e.g. Kong reporting non-conflict error). {% endif_version %} |
+| `ingress_controller_configuration_push_count[success=true|false][protocol=db-less|deck]` | Count of successful or failed configuration pushes to Kong. <br><br> `protocol` describes the configuration protocol in use, which can be `db-less` or `deck`. <br><br> `success` logs the status of configuration updates. If `success` is `false`, an unrecoverable error occurred.  If `success` is `true`, the push succeeded with no errors. |
```

Comment:
Why would we want to have 2 definitions here? I believe the one with `failure_reason` is enough. Especially this doesn't make sense when you have the docs opened at KIC version lower than 2.7: then those are basically the same.

Reply:
Good catch 😬 [74e6fbb](https://github.com/Kong/docs.konghq.com/pull/4500/commits/74e6fbb09a0d5b242618b43cbb90a3f7e49937cb)